backend/seed_data/badges.yaml

```
@@ -3,14 +3,20 @@ badges:
   - title: "Hash Maps"
     description: "Awarded for completing the hash tables unit."
     type: CONTENT
-    criteria_expression: "user_unit.completed == True and user_unit.order == 1"
+    criteria_expression: "user_unit_completed == True and unit_title == 'Hash Maps'"
     event_type: UNIT_COMPLETION # NOTE: new challenge, need to convert enums
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)